__tests__/spelling.js

```diff
@@ -9,10 +9,21 @@ const reporter = require('vfile-reporter')
 
 const describeRule = require('../test-utils/describe-rule')
 
-const ignore = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'))
+const ignoreWords = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'))
+// https://www.w3.org/WAI/WCAG21/Techniques
+const ignoreTechniques = [`ARIA`, `C`, `F`, `G`, `H`].reduce((out, techniquePrefix) => {
```

Comment:
Some prefixes are missing (FLASH, SCR, …) but I agree that they shouldn't concern us immediately and can be added if needed…